common/display/displayplanemanager.cpp

```diff
@@ -518,7 +519,7 @@ void DisplayPlaneManager::EnsureOffScreenTarget(DisplayPlaneState &plane) {
 
   if (!surface) {
     NativeSurface *new_surface = NULL;
-    if (video_separate) {
+    if (video_separate && !force_normal_surface) {
```

Comment:
Set (force_normal_surface=false) here?  Or does this flag remain in the same state?